### PR TITLE
Update image versions in emptyDir e2e

### PR DESCRIPTION
**Pull request by pmorie**
@kubernetes/goog-testing

### CODE REVIEW

File: test/e2e/empty_dir.go [hunk 1]
```diff
@@ -29,8 +29,8 @@ import (
 )
 
 const (
-	testImageRootUid    = "gcr.io/google_containers/mounttest:0.4"
-	testImageNonRootUid = "gcr.io/google_containers/mounttest-user:0.2"
+	testImageRootUid    = "gcr.io/google_containers/mounttest:0.5"
+	testImageNonRootUid = "gcr.io/google_containers/mounttest-user:0.3"
```
Comment (ixdy) [inline test/e2e/empty_dir.go]: it looks like a mounttest-user:0.3 hasn't been pushed yet.

**Comment by ixdy** [comment]
```console
$ make push
sudo docker build -t gcr.io/google_containers/mounttest-user:0.3 .
Sending build context to Docker daemon 3.584 kB
Step 0 : FROM gcr.io/google_containers/mounttest:0.5
 ---> 2e96bc32211b
Step 1 : USER 1001
 ---> Using cache
 ---> 3f2f373c89b7
Successfully built 3f2f373c89b7
gcloud docker push gcr.io/google_containers/mounttest-user:0.3
The push refers to a repository [gcr.io/google_containers/mounttest-user] (len: 1)
3f2f373c89b7: Image successfully pushed 
2e96bc32211b: Image successfully pushed 
9fc8c69b350f: Image successfully pushed 
0.3: digest: sha256:dc089162fcb9e02a31f077672523f192e4452676d94411f01293624df7df3e1f size: 4275
```

@k8s-bot test this again please

**Comment by pmorie** [comment]
Thanks a lot @ixdy

**Comment by ixdy** [comment]
LGTM

**Comment by ixdy** [comment]
@k8s-bot unit test this please

**Comment by wojtek-t** [comment]
@k8s-bot unit test this please

**Comment by ixdy** [comment]
@k8s-bot unit test this please

**Pull request merged**